src/main/java/gov/cabinetoffice/gap/applybackend/service/GrantMandatoryQuestionService.java

```diff
@@ -38,7 +38,11 @@ public GrantMandatoryQuestions getGrantMandatoryQuestionById(UUID id, String app
         return grantMandatoryQuestion.get();
     }
 
-    public GrantMandatoryQuestions getGrantMandatoryQuestionBySubmissionId(UUID submissionId, String applicantSub) {
+    /*
+        TODO I think we should decouple access control and CRUD functionality.
+        These methods should not require an applicant ID to be passed in to confirm ownership.  
+     */
+    public GrantMandatoryQuestions getGrantMandatoryQuestionBySubmissionIdAndApplicantSub(UUID submissionId, String applicantSub) {
```

Comment:
Agreed, we do this elsewhere don't we by getting the ID from the Principal I think?